src/components/Dialog/index.test.js

```diff
@@ -44,7 +43,5 @@ describe('Dialog', () => {
     expect(window.getComputedStyle(screen.getByRole('dialog'))['max-width']).toBe(
       '1000px'
     );
-    // Scrolling should be locked when dialog is open
-    expect(window.getComputedStyle(document.documentElement).overflow).toBe('hidden');
```

Comment:
To my surprise (and actually when I think about, this is smart) Reach UI Dialog does not apply overflow hidden to the body, but merely locks the scrolling on body. The most powerful part of this is not having to deal with a shifting page as a scrollbar enters and leaves (our current implementation) or, what most implementations do, try to sniff the width of the scrollbar and add commensurate right padding to the screen so that things don't shift around.